src/roles/vstat-deploy/tasks/main.yml

```diff
@@ -359,16 +359,22 @@
             data_group_params: " -d {{ groups['data_vstats'][0] }},{{ groups['data_vstats'][1] }},{{ groups['data_vstats'][2] }}"
           when: stats_out | default(false)
 
-        - name: Execute VSTAT cluster script on standalone or clustered vsds
+        - name: Execute VSTAT cluster script on standalone or clustered vsds (ignoring errors)
           command: /opt/vsd/vsd-es-cluster-config.sh -e {{ groups['vstats'][0] }},{{ groups['vstats'][1] }},{{ groups['vstats'][2] }}{{ data_group_params | default('') }}
           environment:
             SSHPASS: "{{ vstat_default_password }}"
-          no_log: "{{ lookup('env', 'METROAE_NO_LOG') or 'true' }}"
```

Comment:
we need this for not showing password in logs

Reply:
ah I didn't mean to remove that